clock/scheduledJobs/deactivateRooms.js

```diff
@@ -11,6 +11,12 @@
  *
  * Gets an array of all rooms which should be deactivated, then deactivates
  * them and does a sync if this is enabled in the room settings.
+ *
+ * Note:
+ * Deactivate room was removed from the job in Refocus because now
+ * GUS/ORG62 must be checked before deactivating. This work has been
```

Comment:
can't refer to these private resources in public github - can you state this more generically?

Reply:
👍🏻